examples/ArduinoUnoNano-basic-Australia/ArduinoUnoNano-basic-Australia.ino

```diff
@@ -140,7 +137,7 @@ void loop()
     led_on();
 
     Serial.println("TXing");
-    myLora.tx("Oi!"); // one byte, blocking function
+    myLora.tx("!"); // one byte, blocking function
```

Comment:
I liked the "Oi!" ;)
You could've just changed the comment to say "three bytes," too.